RecoLocalFastTime/FTLRecProducers/python/mtdRecHits_cfi.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoLocalFastTime.FTLCommonAlgos.mtdRecHitAlgo_cff import mtdRecHitAlgo
+
+_barrelAlgo = mtdRecHitAlgo.clone()
+_endcapAlgo = mtdRecHitAlgo.clone()
```

Comment:
It is an awkward design to have a copy of the algo for each subdetector, while each copy contains parameters for both subdetectors. Consider removing the separate BTL/ETL functions from the algo and just have one function, controlled by parameters specified for one subdetector.

Reply:
Done